packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SubjectGroupViewer/SubjectGroupViewerContainer.js

```diff
@@ -238,9 +308,16 @@ class SubjectGroupViewerContainer extends React.Component {
       return null
     }
 
+    // Note: the Task's Annotations are initialised by the SubjectGroupTask component.
+    const annotation = toJS(classification?.annotation(currentTask))
```

Comment:
Problem: changes to annotations aren't being observed properly.

- Annotations are changed by the SGVGridCell calling it's parent's parent's `SubjectGroupViewerContainer.toggleCellAnnotation()` function.
- The stores are updated correctly (i.e. if I submit a Classification, the annotations are correct). 
- However, the visuals don't reflect changes to the annotation unless another change is triggered (e.g. switching the Subject Viewer from annotate mode to view mode)
- Basically, the `SubjectGroupViewerContainer`'s `render()` method doesn't trigger when the annotation changes.

Solution: `toJS()` somehow works.

Solution's Problem: we generally try to avoid using `toJS()` in our code.

Other solutions tried:

- Wrapping `SubjectGroupViewer` in `observer`. Result: click-to-annotate causes the correct visual change, but drag-to-pan function breaks.
- Wrapping `SGVGridCell` in `observer`. Result: no change

Other solutions considered:

- Wrapping the `annotation` in an `observable`?

Reply:
My initial thoughts: The appearance of `SVGGridCell` should reflect its component state (selected/not selected), which is toggled by clicking on it. Think of the grid cell as being a HTML `checkbox` element. Changes in component state are passed to the annotation value via `annotation.update`.

Reply:
I think the grid cell itself can have `role="checkbox" aria-checked=(true|false)`.